src/Preview.tsx

```diff
@@ -229,6 +229,13 @@ const Preview: React.FC<PreviewProps> = props => {
     ],
   );
 
+  const onDoubleClick: React.MouseEventHandler<HTMLBodyElement> = () => {
+    if (visible) {
+      setScale(1);
+      setPosition(initialPosition);
+    }
+  };
```

Comment:
```suggestion
  const onDoubleClick: React.MouseEventHandler<HTMLBodyElement> = () => {
    if (!visible) {
      return;
    }
    setScale(1);
    setPosition(initialPosition);
  };
```

Reply:
@afc163 Thanks for the quick review! Fixed!